Add missing meta descriptions (#235)
mongodb/docs-php-library: source/tutorial/commands.txt

Execute Database Commands
=========================

.. meta::

>    :description: Learn to execute database commands in PHP using the MongoDB PHP Library, including handling single and multiple result documents and setting custom read preferences.

Reviewer: ```suggestion
   :description: Learn to execute database commands with the MongoDB PHP Library, including handling single and multiple result documents and setting custom read preferences.
```
